e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -0,0 +1,543 @@
+            //.withScaledService("api", 2) // failing now since it's not changing ports
+            //.withScaledService("worker", 2)
+            .withExposedService("db", 5432)
+            .withExposedService("api", 8080);
+
+    private static HttpClient httpClient;
+
+    private static DriverManagerDataSource dataSource;
+
+    private static String AB2D_API_URL = "http://localhost:8080/api/v1/fhir/";
+
+    private static String AB2D_ADMIN_URL = "http://localhost:8080/api/v1/admin/";
+
+    private static final String PATIENT_EXPORT_PATH = "Patient/$export";
+
+    private static String jwtStr = null;
+
+    private static String TEST_USER;
+
+    private static final long DEFAULT_TIMEOUT = 30;
+
+    private static final int THREAD_DELAY = 20000;
+
+    private Integer sponsorId = null;
+
+
+    @BeforeAll
+    public void init() throws IOException, InterruptedException, JSONException {
+        container.start();
+
+        Yaml yaml = new Yaml();
+        InputStream inputStream = getClass().getResourceAsStream("/default-config.yml");
+        Map<String, String> yamlMap = yaml.load(inputStream);
+        String oktaUrl = yamlMap.get("okta-url");
+        TEST_USER = yamlMap.get("username");
+        String password = yamlMap.get("password");
+        AB2D_API_URL = yamlMap.get("ab2d-api-url");
+        AB2D_ADMIN_URL = yamlMap.get("ab2d-admin-url");
+
+        httpClient = HttpClient.newBuilder()
+                .version(HttpClient.Version.HTTP_1_1)
+                .followRedirects(HttpClient.Redirect.NORMAL)
+                .build();
+
+        var jwtRequestParms = new HashMap<>() {{
+            put("grant_type", "password");
+            put("username", TEST_USER);
+            put("password", password);
+            put("scope", "openid");
+        }};
+
+        HttpRequest jwtRequest = HttpRequest.newBuilder()
+                .uri(URI.create(oktaUrl))
+                .timeout(Duration.ofSeconds(DEFAULT_TIMEOUT))
+                .header("Content-Type", "application/x-www-form-urlencoded")
+                .header("Accept", "application/json")
+                .header("Authorization", "Basic MG9hMXB4YXB6ZE9XaUhmOXUzNTc6WGIxQnpDb1ktS0Y4Z1FhTF9JMmVzVmxsdkpZZ2VzX2d1QTVHZTlpaQ==")
+                .POST(buildFormDataFromMap(jwtRequestParms))
+                .build();
+
+        HttpResponse<String> jwtResponse = httpClient.send(jwtRequest, HttpResponse.BodyHandlers.ofString());
+        String responseJwtString = jwtResponse.body();
+        JSONObject responseJsonObject = new JSONObject(responseJwtString);
+        jwtStr = responseJsonObject.getString("access_token");
+
+        dataSource = new DriverManagerDataSource();
+        dataSource.setDriverClassName("org.postgresql.Driver");
+        dataSource.setUrl("jdbc:postgresql://localhost:5432/ab2d");
+        dataSource.setUsername("ab2d");
+        dataSource.setPassword("ab2d");
+
+        uploadOrgStructureReport();
+        uploadAttestationReport();
```

Comment:
This is going to be complicated because the test user (EileenCFrierson@example.com) does not have the admin rights.

I think initially the e2e test might need to assume the only data in the database is whatever is populated by the `seed.sql`.